src/shared/pprof/pprof.cc

```diff
@@ -16,24 +16,21 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-#include "src/stirling/source_connectors/perf_profiler/pprof/pprof.h"
-
-#include <gflags/gflags.h>
+#include "src/shared/pprof/pprof.h"
 
 #include <absl/strings/str_split.h>
 #include <vector>
 
-DECLARE_uint32(stirling_profiler_stack_trace_sample_period_ms);
-
 namespace px {
-namespace stirling {
+namespace shared {
 
-PProfProfile CreatePProfProfile(const uint32_t num_cpus, const histo_t& histo) {
+PProfProfile CreatePProfProfile(const uint32_t num_cpus, const uint32_t period_ms,
+                                const histo_t& histo) {
```

Comment:
nit: should follow style guide for type names https://google.github.io/styleguide/cppguide.html#Type_Names. I know this isn't really followed in `src/stirling` but we do follow it in `src/shared`

Reply:
Removed the `histo_t` alias and replaced it directly with the underlying.